src/plugins/opensearch_dashboards_react/public/validated_range/validated_dual_range.tsx

```diff
@@ -88,7 +88,7 @@ export class ValidatedDualRange extends Component<Props> {
   // @ts-ignore state populated by getDerivedStateFromProps
   state: State = {};
 
-  _onChange = (value: Value) => {
```

Comment:
I'm not able to change input control here:
```
http://localhost:5601//app/visualize#/create?type=input_control_vis&_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:now-15m,to:now))&_a=(filters:!(),linked:!f,query:(language:kuery,query:''),uiState:(),vis:(aggs:!(),params:(controls:!((fieldName:hour_of_day,id:'1644025754232',indexPattern:d3d7af60-4c81-11e8-b3d7-01146121b73d,label:'',options:(decimalPlaces:0,step:1),parent:'',type:range)),pinFilters:!f,updateFiltersOnChange:!f,useTimeFilter:!f),title:'',type:input_control_vis))
```

Reply:
Yeah not sure why this was made private. Updating that now.

Reply:
Turns out all of this was caused by discrepancies with the `@types/react` package in the yarn.lock file. I resolved that and now this change is not necessary.